docs/csharp/codedoc.md

```diff
@@ -14,23 +14,38 @@ ms.assetid: 8e75e317-4a55-45f2-a866-e76124171838
 
 # Documenting your code
 
-XML documentation comments are a special kind of comment, added above the definition of any user defined type or member. 
+XML documentation comments are a special kind of comment, added above the definition of any user-defined type or member. 
 They are special because they can be processed by the compiler to generate an XML documentation file at compile time.
-The compiler generated XML file can be distributed alongside your .NET assembly so that Visual Studio and other IDEs can show quick information about types or members when performing intellisense.
-Additionally the XML file can be run through tools like [DocFX](https://dotnet.github.io/docfx/) and [Sandcastle](https://github.com/EWSoftware/SHFB) to generate full on API reference websites.
+The compiler generated XML file can be distributed alongside your .NET assembly so that Visual Studio and other IDEs can use IntelliSense to show quick information about types or members. Additionally, the XML file can be run through tools like [DocFX](https://dotnet.github.io/docfx/) and [Sandcastle](https://github.com/EWSoftware/SHFB) to generate API reference websites.
 
-XML documentation comments like all other comments are ignored by the compiler, to enable generation of the XML file add `"xmlDoc":true` under `buildOptions` in your `project.json` when using .NET Core or use the `/doc` compiler option for the .NET framework.
-See the [/doc](https://msdn.microsoft.com/library/3260k4x7.aspx) article on MSDN to learn how to enable XML documentation generation in Visual Studio.
+XML documentation comments, like all other comments, are ignored by the compiler.
 
-XML documentation comments are characterized by triple forward slashes (`///`) and an XML formatted comment body.
+You can generate the XML file at compile type by doing one of the following:
+
+- If you are developing an application with .NET Core from the command line, you can add a [DocumentationFile element](http://docs.microsoft.com/visualstudio/msbuild/common-msbuild-project-properties) to the `<PropertyGroup>` section of your .csproj project file. The following example generates an XML file in the project directory with the same root filename as the project:
+
+   ```xml
+   <DocumentationFile>$(MSBuildProjectName).xml</DocumentationFile>
+   ```
+
+   You can also specify the exact absolute or relative path and name of the XML file. The following example generates the XML file in the same directory as the debug version of an application:
+
+    ```xml
+   <DocumentationFile>bin\Debug\netcoreapp1.0\App.xml</DocumentationFile>
+   ```
+
+-- If you are developing an application using Visual Studio, right-click on the project and select **Properties**. In the properties dialog, select the **Build** tab, and check **XML documentation file:**. You can also change the location to which the compiler writes the file. 
```

Comment:
remove one of the dashes
style guide: If a command label ends with a colon or an ellipsis, don’t include that end punctuation in instructions.